remoteworkitem/remoteworkitem.go

```diff
@@ -11,16 +11,43 @@ const (
 	SystemTitle        = "system.title"
 	SystemDescription  = "system.description"
 	SystemStatus       = "system.status"
-	ProviderGithub     = "github"
-	ProviderJira       = "jira"
+	SystemAssignee     = "system.assignee"
+	SystemCreator      = "system.creator"
+
+	GithubTitle       = "title"
```

Comment:
I'm guessing, that this is the *path* to the title field in github HTTP response body, right? Please document these constants a bit. Not every single one of them but a just an overall comment would suffice, I guess.